sample_data_config/ejp-xml-data-pipeline.config.yaml

```diff
@@ -0,0 +1,12 @@
+gcpProjectName: 'elife-data-pipeline'
+dataPipelineId: ejp_xml_pipeline_id
+dataset: '{ENV}'
+manuscriptTable: manuscript_table_test
```

Comment:
I understand this is a sample config file but are these resembling the desired table names?